lemur/certificates/service.py

```diff
@@ -150,7 +151,7 @@ def get_all_certs_attached_to_endpoint_without_autorotate():
         """
     return (
         Certificate.query.filter(Certificate.endpoints.any())
-        .filter(Certificate.rotation == False)
+        .filter(Certificate.rotation == false())
```

Comment:
lint does not like `X == boolean` comparision, which is needed in filter. Thus using `sqlalchemy.sql.expression`